storybook/README.md

```diff
@@ -50,33 +50,15 @@ Once started, you can access Storybook at: http://localhost:6006/
 
 ## Build and start environment with docker
 
-### Install Project Dependencies with Docker
-
-**Install PHP Dependencies**
-
-Run the following command to install PHP dependencies via Docker:
+A Compose file is provided in the `storybook/` folder. It is responsible in installing the dependencies before running PHP and Storybook.
+To start Storybook via Docker, use the following command:
 
 ```shell
-$ docker run -u 1000:1000 --rm --interactive --tty -v "$PWD"/../:/app -w /app/storybook composer install
+$ cd storybook/
```

Comment:
```suggestion
```

We should follow the context of the README, the commands are launched from the directory thereof. As indicated above: # From the storybook/folder

Reply:
```suggestion
# From the storybook/ folder
```